### PR TITLE
feat: updated readme

### PR DESCRIPTION
**Pull request by EinSev**

### CODE REVIEW

**Comment by Mathematti** [comment]
Thanks for updating the readme without updating the readme

File: README.md [hunk 1]
```diff
@@ -1,40 +1,82 @@
...
+* [Acknowledgements](#acknowledgements)
 
-If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has also implemented a [Take Over Mode](https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669) that is more performant. You can enable it by the following steps:
+## About The Project
 
-1. Disable the built-in TypeScript Extension
-    1) Run `Extensions: Show Built-in Extensions` from VSCode's command palette
-    2) Find `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`
-2. Reload the VSCode window by running `Developer: Reload Window` from the command palette.
+JourneyPlanner is an innovative tool that makes planning group trips easier. Developed by 4 dedicated HTL students (vocational school in Austria), this Vue.js and Supabase project is designed to simplify the organization journeys and ensure a smooth experience for everyone involved.
```
Comment (Mathematti) [inline README.md]: grammatical error

File: .github/JP_dashboard_v1.0.png
Comment (Mathematti) [inline .github/JP_dashboard_v1.0.png:1]: Why add this file? Is it used anywhere?

File: README.md [hunk 2]
```diff
@@ -1,40 +1,82 @@
...
+## Roadmap
 
-```sh
-npm run build
-```
+See the [open issues](https://github.com/JourneyPlanner/JourneyPlanner/issues) for a list of proposed features (and known issues).
+
+## License
+
+Distributed under the MIT License. See [LICENSE](https://github.com/JourneyPlanner/JourneyPlanner/blob/main/LICENSE.md) for more information.
```
Comment (Mathematti) [inline README.md]: License has to be discussed with the team

File: .github/JP_dashboard_v1.0.png
Comment (EinSev) [reply .github/JP_dashboard_v1.0.png:1]: Now it is. Forgot to add it in the final readme.

**Review by Mathematti — APPROVED** [review]

**Pull request merged**